sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -81,7 +81,7 @@ public partial class AzureDeveloperCliCredentialOptions : Azure.Identity.TokenCr
     public partial class AzurePipelinesCredential : Azure.Core.TokenCredential
     {
         protected AzurePipelinesCredential() { }
-        public AzurePipelinesCredential(string tenantId, string clientId, string serviceConnectionId, Azure.Identity.AzurePipelinesCredentialOptions options = null) { }
+        public AzurePipelinesCredential(string systemAccessToken, string clientId = null, string tenantId = null, string serviceConnectionId = null, Azure.Identity.AzurePipelinesCredentialOptions options = null) { }
```

Comment:
Q: is it common to have optional positional arguments? Why not add them to the AzurePipelinesCredentialOptions instead?

Looks like AzurePipelinesCredentialOptions already defines some of these - why would we have them defined in both places?

Reply:
This makes them more prominent, but I could go either way - they are internal in the options, currently.

Reply:
Whatever we decide here, I think we should be consistent cross-lang. I'm discussing with the .NET architect to get some input and we can decide on an approach after that.

Reply:
I think these should be in the options rather than in the constructor as they are for `WorflowIdentityCredential`. Also is there a chance that `systemAccessToken` becomes a non-required argument in the future?

Reply:
> I think these should be in the options rather than in the constructor as they are for `WorflowIdentityCredential`. 

I'm open to it - let's discuss in standup today

> Also is there a chance that `systemAccessToken` becomes a non-required argument in the future?

I don't believe so